.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- !! Thank your for opening a PR !! -->
+
+<!--- Provide a short summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
+<!--- If applicable, please type an `x` in one of the boxes below and provide the issue number after the # sign: -->
+- [ ] Closes #
+- [ ] Related to #
+
+## Checklist
+<!--- Make sure you have completed the following steps before submitting your PR -->
+<!--- Feel free to type an `x` in all the boxes below to let us know you have completed the steps: -->
+- [ ] Checked that the pre-commit checks pass (see [your-first-code-contribution](https://github.com/Mews/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution) and also [pre-commit](https://docs.pymc.io/en/latest/contributing/python_style.html))
+- [ ] Checked that all the tests pass (see [your-first-code-contribution](https://github.com/Mews/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution))
+- [ ] Checked that your commit messages follow [the specified guidelines](https://github.com/Mews/flask-utils/blob/main/CONTRIBUTING.md#commit-messages)
```

Comment:
```suggestion
- [ ] Checked that the pre-commit checks pass (see [your-first-code-contribution](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution) and also [pre-commit](https://docs.pymc.io/en/latest/contributing/python_style.html))
- [ ] Checked that all the tests pass (see [your-first-code-contribution](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md#your-first-code-contribution))
- [ ] Checked that your commit messages follow [the specified guidelines](https://github.com/Seluj78/flask-utils/blob/main/CONTRIBUTING.md#commit-messages)
```

Reply:
Oops I didn't realize I put the links from my fork